### PR TITLE
Add option to set calendar name and change published ttl

### PR DESCRIPTION
**Pull request by fholzer**
What do you think about supporting this?

Also thinking about adding support for BUSYSTATUS. (https://msdn.microsoft.com/en-us/library/ee219533(v=exchg.80).aspx

### CODE REVIEW

File: test/pipeline/build.spec.js [hunk 1]
```diff
@@ -2,6 +2,16 @@ import { expect } from 'chai'
 import { buildEvent } from '../../src/pipeline'
 
 describe('pipeline.build properties', () => {
+  describe('calendarName', () => {
+    it('removes a falsey value', () => {
+      const event = buildEvent()
+      expect(event.description).not.to.exist
```
Comment (adamgibbons) [inline test/pipeline/build.spec.js]: Should this be checking `event.description` or `event.calendarName`?

File: src/pipeline/format.js [hunk 2]
```diff
@@ -39,7 +41,8 @@ export default function formatEvent(attributes = {}) {
   icsFormat += 'CALSCALE:GREGORIAN\r\n'
   icsFormat += foldLine(`PRODID:${productId}`) + '\r\n'
   icsFormat += foldLine(`METHOD:${method}`) + '\r\n'
-  icsFormat += `X-PUBLISHED-TTL:PT1H\r\n`
+  icsFormat += calendarName ? (foldLine(`X-WR-CALNAME:${calendarName}`) + '\r\n') : ''
+  icsFormat += `X-PUBLISHED-TTL:${ttl ? formatDuration(ttl) : "PT1H"}\r\n`
```
Comment (adamgibbons) [inline src/pipeline/format.js]: `X-PUBLISEHD-TTL` shouldn't appear unless a value is provided.

**Comment by sashagim** [comment]
Would love an option to set the name of the calendar!

**Comment by GabrielTK** [comment]
Sorry to ask, but... Why was this PR closed?

**Comment by adamgibbons** [comment]
@GabrielTK I closed it because the requested changes went unaddressed for seven months.

**Pull request closed without merging**